lib/builder.js

```diff
@@ -124,6 +124,9 @@ Builder.prototype.reset = function(baseLoader) {
   loader.builder = true;
   // put the loader in the builder environment
   loader.config({ build: true });
+  
+  // make builder accessible in plugins
+  loader.systemJSBuilder = this;
```

Comment:
How about making this `loader.builder`? This does collide with the true value above I know, but checks like `if (this.builder)` won't be affected?